core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/seo/components/dot-results-seo-tool/dot-results-seo-tool.component.ts

```diff
@@ -51,4 +78,11 @@ export class DotResultsSeoToolComponent implements OnInit {
             }
         ];
     }
+
+    ngOnChanges() {
+        this.currentResults = this.dotSeoMetaTagsService.getFilteredMetaTagsByMedia(
```

Comment:
If you need to handle 2 inputs at the same time, try using 

```
ngOnChanges(changes: SimpleChanges) {
    console.log(changes)
  }
```
to verify if the **currentValue** if diferent to **undefined**, every time you change the input **ngOnChanges** is triggered.